sql/i.salesOfferPackage.sql

```diff
@@ -0,0 +1,18 @@
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+
+DROP TABLE IF EXISTS salesOfferPackage;
+
+CREATE TABLE salesOfferPackage(
+    `id` int(11) NOT NULL AUTO_INCREMENT,
+    `nocCode` varchar(255) NOT NULL,
+    `name` varchar(255) DEFAULT NULL,
+    `description` varchar(255) DEFAULT NULL,
+    `purchaseLocation` varchar(255) DEFAULT NULL,
+    `paymentMethod` varchar(255) DEFAULT NULL,
+    `ticketFormat` varchar(255) DEFAULT NULL,
+    INDEX idx_nocCode (nocCode),
+    CONSTRAINT fk_salesOfferPackage_nocTable_nocCode FOREIGN KEY (nocCode) REFERENCES nocTable(nocCode),
```

Comment:
We may need to get rid of this constraint actually since it won't work with the IWBusCo NOC, it doesn't give much benefit anyway really since we're not joining the two tables